module/zfs/dmu_traverse.c

```diff
@@ -587,7 +587,7 @@ traverse_prefetch_thread(void *arg)
  */
 static int
 traverse_impl(spa_t *spa, dsl_dataset_t *ds, uint64_t objset, blkptr_t *rootbp,
-    uint64_t txg_start, zbookmark_phys_t *resume, int flags,
+    uint64_t txg_start, zbookmark_phys_t *resume, uint32_t flags,
```

Comment:
Looks like this should be `arc_flags_t flags`, not a `uint32_t`.

Reply:
That will propagate across the entire module.